File: gitb-ui/app/models/ConformanceStatusItem.scala
package models

import java.sql.Timestamp

case class ConformanceStatusItem(
  testSuiteId: Long, testSuiteName: String, testSuiteDescription: Option[String], testSuiteHasDocumentation: Boolean, testSuiteSpecReference: Option[String], testSuiteSpecDescription: Option[String], testSuiteSpecLink: Option[String],
  testCaseId: Long, testCaseName: String, testCaseDescription: Option[String], testCaseHasDocumentation: Boolean, testCaseSpecReference: Option[String], testCaseSpecDescription: Option[String], testCaseSpecLink: Option[String],
  result: String, outputMessage: Option[String], sessionId: Option[String], sessionTime: Option[Timestamp],
  testCaseOptional: Boolean, testCaseDisabled: Boolean, testCaseTags: Option[String]
)

File: gitb-ui/ui/src/app/types/conformance-status-item.ts
export interface ConformanceStatusItem {

    id?: number
    exportPending?: boolean
    actionPending?: boolean
    
    testSuiteId: number
    testSuiteName: string
    testSuiteDescription?: string
    testSuiteHasDocumentation: boolean
    testCaseId: number
    testCaseName: string
    testCaseDescription?: string
    testCaseHasDocumentation: boolean
    result: string
    outputMessage?: string
    sessionId?: string
    sessionTime?: string
    testCaseOptional: boolean
    testCaseDisabled: boolean

}
